src/gwt/src/org/rstudio/studio/client/common/SimpleRequestCallback.java

```diff
@@ -65,7 +65,7 @@ public void onError(ServerError error)
                message = errMsg.stringValue();
          }
       }
-      
+     
```

Comment:
nit: unnecessary whitespace change

Reply:
Unless I also made that change in main, in which case it's fine :-)

Reply:
I probably put that in at some point when I was experimenting, and I generally prefer whitespace after closing brackets. I can take this back out.